src/components/AddItem.js

```diff
@@ -1,19 +1,32 @@
 import React, { useState } from 'react';
 import { writeToFirestore } from '../lib/firebase';
 
-const AddItem = ({token}) => {
+const AddItem = ({ token, results }) => {
   let [name, setName] = useState('');
   let [frequency, setFrequency] = useState(7);
 
   function handleSubmitForm(event) {
     event.preventDefault();
     if (name.length > 0) {
-      writeToFirestore(token, {
-        name,
-        frequency,
-        lastPurchaseDate: null,
+      let count = 0;
```

Comment:
Not sure about how to clean this up, as it looks a little messier than it should be. Feedback very welcome on this.